holoviews/element/path.py

```diff
@@ -191,24 +203,52 @@ class Ellipse(BaseShape):
 
     y = param.Number(default=0, doc="The y-position of the ellipse center.")
 
-    height = param.Number(default=1, doc="The height of the ellipse.")
+    height = param.Number(default=1, doc="""
+       The height of the ellipse. Will be deprecated in HoloViews 2.0:
+       use the major_axis_length parameter instead.""")
+
+    major_axis_length = param.Number(default=1, doc="The length of the ellipse major axis")
+
+    minor_axis_length = param.Number(default=1, doc="The length of the ellipse minor axis")
 
-    aspect= param.Number(default=1.0, doc="The aspect ratio of the ellipse.")
+    aspect= param.Number(default=1.0, doc="""
+       Final multipler only applied to the minor axis length. Allows
+       ellipses to be specified without the (major axis length, minor
+       axis length) tuple format.""")
 
-    orientation = param.Number(default=0, doc="Orientation in the Cartesian coordinate system, the counterclockwise angle in radian between the first axis and the horizontal.")
+    orientation = param.Number(default=0, doc="""
+       Orientation in the Cartesian coordinate system, the
+       counterclockwise angle in radians between the first axis and the
+       horizontal.""")
 
     samples = param.Number(default=100, doc="The sample count used to draw the ellipse.")
 
     group = param.String(default='Ellipse', constant=True, doc="The assigned group name.")
 
-    def __init__(self, x, y, height, **params):
-        super(Ellipse, self).__init__([], x=x, y=y, height=height, **params)
+    def __init__(self, x, y, spec, **params):
+
+        if 'height' in params:
+            spec = params['height']
+
+        if isinstance(spec, tuple):
+            (major, minor) = spec
+            if major < minor:
+                raise ValueError('Minor axis length must be less than major axis length')
+            params['height'] = major
+        else:
```

Comment:
Is it necessary to enforce this restriction?  It's confusing if someone sets them the wrong way around, but e.g. if there are widgets, it can be tough to ensure that it doesn't get flipped in that way.  You could even consider labelling them axis1 and axis2 for that reason, and *suggesting* that people use axis1 for the major axis.